storm-server/src/main/java/org/apache/storm/daemon/drpc/DRPC.java

```diff
@@ -208,6 +208,7 @@ public void failRequest(String id, DRPCExecutionException e) throws Authorizatio
         }
     }
 
+    //Should this be changed to private?
```

Comment:
If no one is calling it from outside go ahead.

Reply:
It is not called publicly, but this looks like an asynchronous method which may be used later.

Reply:
If we need it later we can make it public then